src/core/cita_bft.rs

```diff
@@ -1033,11 +1035,23 @@ impl TenderMint {
                     pubkey_to_address(&pubkey)
                 );
 
+                // verify the timestamp
+                let block_timestamp = block.get_header().timestamp;
```

Comment:
Verify the timestamp iff `EconomicalModel::Charge`

Reply:
Is there any good way to fetch EconomicalModel instead of sending a request to cita-executor?

Reply:
No.Fetch the consensus config when the consensus service starts? The init-config shoud include block-interval, economical-model.